src/models/multiBarChart.js

```diff
@@ -429,8 +674,25 @@ nv.models.multiBarChart = function() {
             rightAlignYAxis = _;
             yAxis.orient( rightAlignYAxis ? 'right' : 'left');
         }},
+        xTickFormat: {get: function(){return xAxis.xTickFormat();}, set: function(_){
```

Comment:
Why were these options created?  Normally we just use xAxis.tickFormat().  Though, I think the goal here is to set both the chart and the focus chart's axis formatting